Exercises-1-Basic/src/exercises/Num1.java

```diff
@@ -0,0 +1,12 @@
+package exercises;
+//Write a program to print numbers from 1 to 10. 
+public class Num1 {
+
+    public static void main(String[] args) {
+	int i = 1;
+	for (i = 1; i <= 10; i++) {
+	    System.out.println(i);
+	}
```

Comment:
```suggestion
	for (int i = 1; i <= 10; i++) {
	    System.out.println(i);
	}
```
It is clearer to put I declaration inside the `for`

Reply:
Done